pkg/object/globalfilter/globalfilter.go

```diff
@@ -0,0 +1,217 @@
+package globalfilter
```

Comment:
Please add license header in this file. The `Copyright` stuff to pass License checker.